src/fontra/views/editor/editor.js

```diff
@@ -1005,26 +1009,38 @@ export class EditorController {
   async showDialogGlyphEditCannotEditReadOnly(create = false) {
     const glyphName = this.sceneSettings.selectedGlyphName;
     await message(
-      `Can’t ${create ? "create" : "edit"} glyph “${glyphName}”`,
-      "The font is read-only."
+      create
```

Comment:
Suggest to do the switching inside the `translate()` call, something like this:
```js
   translate(create ? key1 : key2, glyphName)
```

Reply:
done.